build/rewrite.go

```diff
@@ -997,3 +1002,18 @@ func reorderArguments(f *File, info *RewriteInfo) {
 		}
 	})
 }
+
+// editOctals inserts 'o' into octal numbers to make it more obvious they are octal
+// 0123 -> 0o123
+func editOctals(f *File, info *RewriteInfo) {
+	Walk(f, func(expr Expr, stack []Expr) {
+		l, ok := expr.(*LiteralExpr)
+		if !ok {
+			return
+		}
+		if len(l.Token) > 1 && l.Token[0] == '0' && l.Token[1] != 'o' && l.Token[1] != 'x' {
```

Comment:
This may fail with uppercase characters, e.g. `0X12`.
I suggest reverting the condition: test if the 2nd character is a digit.

Reply:
Done